chainer/utils/array.py

```diff
@@ -35,3 +36,16 @@ def empty_like(x):
         return cuda.cupy.empty_like(x)
     else:
         return numpy.empty_like(x)
+
+
+def sum_to(x, shape):
```

Comment:
Could you assert `not isinstance(x, Variable)`?  Once `Variable.sum` is implemented, `chainer.utils.sum_to` can compute `Variable`'s "sum_to" but its backward is slower than `F.sum_to`'s.

Reply:
Fixed to check if x is not Variable.